### ISSUE TITLE
Git push broken when cpu limits are set on application

### ISSUE THREAD

**Issue by wvdeutekom** [deis/workflow#452]
*Running Deis Workflow 2.3 on GKE 1.2.5*

When I limit a (docker) ruby application on CPU using the following command:
```
deis limits:set -a core-api-development cmd=200m -c
```

After setting those limits I get the following error when git pushing to the deis remote:

```
Counting objects: 1, done.
Writing objects: 100% (1/1), 187 bytes | 0 bytes/s, done.
Total 1 (delta 0), reused 0 (delta 0)
remote: 2016/08/15 13:11:22 Error running git receive hook [getting app config for core-api-development (json: cannot unmarshal string into Go value of type int)]
To ssh://git@deis-builder......:2222/core-api-development.git
 ! [remote rejected] 0af57a8d9779ba08a1b8c9fa6dae441030393fc9 -> master (pre-receive hook declined)
error: failed to push some refs to 'ssh://git@deis-builder......:2222/core-api-development.git'
```

I checked the application's replication controller which notes the following:
```
         resources:
           limits:
             cpu: 200m
```

Removing the cpu limits from the app or replication controller seems to make this bug disappear again.

**Comment by kmala** [comment]
this is fixed and should be working in the upcoming release which is gonna happen this week.
https://github.com/deis/builder/pull/405.
For now as a work around you can do git push and do a set of the limits.

**Comment by wvdeutekom** [comment]
Great, thanks for following this up @kmala!

**Comment by Joshua-Anderson** [comment]
This has been fixed in master and v2.4.0 should be coming out shortly. Thanks for reporting!
